core/src/components/datetime-button/test/overlays/datetime-button.e2e.ts

```diff
@@ -0,0 +1,180 @@
+import { expect } from '@playwright/test';
+import type { Locator } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+import type { EventSpy } from '@utils/test/playwright';
+
+/**
+ * This behavior does not vary across directions.
+ */
+configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
```

Comment:
Tests were previously running for all configs, but they did not need to be. As a result, I limited the tests to only run in LTR direction